cmd/cloudflared/service_template.go

```diff
@@ -124,8 +124,9 @@ func openFile(path string, create bool) (file *os.File, exists bool, err error)
 	if err != nil {
 		return nil, false, err
 	}
+	var fileInfo os.FileInfo
 	if create {
-		fileInfo, err := os.Stat(expandedPath)
+		fileInfo, err = os.Stat(expandedPath)
```

Comment:
This is a good catch, but we need to check if this returns an error and return it, otherwise line 133 will override the error.